shared/nav.js.flow

```diff
@@ -17,6 +17,7 @@ export type Props = {
   routeSelected: Tab,
   routeStack: RouteRenderStack,
   hideNav: boolean,
+  reachability: ?Reachability,
```

Comment:
There is actually an unknown reachability enum (0). I'd make it required and default to Reachability_UNKNOWN.